ConcernsCaseWork/ConcernsCaseWork.CypressTests/cypress/integration/casework-regression/close-case-with-actions.spec.ts

```diff
@@ -334,4 +441,33 @@ describe("Testing closing of cases when there are case actions and concerns", ()
             ClosedCasePage.getClosedCase(caseId);
         });
     }
+
+    function verifyClosedCaseDetails()
+    {
+        Logger.Log("Validate Closed Case row has correct details");
+        caseworkTable
+                .getRowByCaseId(caseId)
+                .then((row) =>
+                {
+                    row
+                        .hasCaseId(caseId)
+                        .hasCreatedDate(toDisplayDate(now))
+                        .hasClosedDate(toDisplayDate(now))
+                        .hasTrust(trustName)
+                        .hasConcern("Governance and compliance: Compliance")
+                })
+
+        Logger.Log("Validate Closed Case has correct details");
+        ClosedCasePage.getClosedCase(caseId).click();
```

Comment:
Can you use the row to invoke this? I think on the case action example it calls select